### PR TITLE
redirect to access-denied when users do not have permissions

### PR DESCRIPTION
**Pull request by dsnyder-fairway**

### CODE REVIEW

File: common-ngx/src/security/authorization.can-activate.ts [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
+              private authorizationService: AuthorizationService,
+              private router: Router) {
   }
 
   canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<boolean> | Promise<boolean> | boolean {
-    return this.authorizationService.hasAnyPermission(route.data[ 'permissions' ]);
+    return this.authorizationService.hasAnyPermission(route.data[ 'permissions' ]).map((permission) => {
+        if (!permission) {
+          this.router.navigate([ this.accessDeniedRoute ]);
+        }
```
Comment (esotrope) [inline common-ngx/src/security/authorization.can-activate.ts]: is this different than the `routing-authorization.can-activate.ts` thing in common?

Comment (dsnyder-fairway) [reply common-ngx/src/security/authorization.can-activate.ts]: yeah, that injectable checks that a user has a non-empty set of permissions. This is used to check to see if a user has a permission belonging to whatever the permissions are for a path

Comment (esotrope) [reply common-ngx/src/security/authorization.can-activate.ts]: gotcha cool. my only question is - what did this do before when we didn't route?? x D

Comment (dsnyder-fairway) [reply common-ngx/src/security/authorization.can-activate.ts]: It would show the navbar and the footer, and that was about it

Comment (dsnyder-fairway) [reply common-ngx/src/security/authorization.can-activate.ts]: @esotrope it looks like this change totally busted reporting 😞

**Pull request merged**